docs/workstation_release_management.rst

```diff
@@ -219,8 +139,8 @@ RPM package
 12. Upon merge to master, ensure that changes deploy to
     ``yum.securedrop.org`` without issue.
 
-``qubes-template-securedrop-workstation`` release and promotion to production
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Release ``qubes-template-securedrop-workstation``
```

Comment:
I think some back ticks are missing around: `(rpm –resign )` a few lines below (step 6), and the two `--` dashes became a longer dash.